etc/helm/pachyderm/templates/pachd/deployment.yaml

```diff
@@ -307,6 +307,15 @@ spec:
               name: {{ default ("pachyderm-console-secret") .Values.console.config.oauthClientSecretSecretName  }}
               key: OAUTH_CLIENT_SECRET
         {{- end }}
+        {{- if .Values.determined.enabled }}
```

Comment:
I think this needs to match the same logic that is in determined/secret.yaml. Pachd can't start otherwise if determined is enabled non-enterprise: `Error: secret "det-oidc-client-secret" not found`